editor/src/app/sites/sections/entries/entries-state/section-entries.state.ts

```diff
@@ -18,6 +22,25 @@ export class SectionEntriesState implements NgxsOnInit {
     });
   }
 
+  @Action(RenameSectionEntriesAction)
+  renameSectionEntries({ setState, getState }: StateContext<SectionEntriesStateModel>, action: RenameSectionEntriesAction) {
+    const state = getState();
+    const newState = cloneDeep(state);
+
+    Object.keys(newState).map(siteName => {
```

Comment:
If you don't need to transform an object into an array don't use the `map` function here. For iterating over objects, use `for ... in`.